### PR TITLE
[tests-only] retag reva issue 249 with the correct issue tags

### LINKED ISSUE #249: Cannot get information of a specific share by providing share_id
## In OC10:

1. Create users `Alice`, `Brian` and `Carol`
2. User `Alice` creates folder `Alice-folder`
3. User `Alice` creates folder `Alice-folder/folder2`
4. User `Carol` creates folder `Carol-folder`
5. User `Alice` shares folder `Alice-folder` with user `Brian` with permissions `all`
6. User `Carol` shares folder `Carol-folder` with user `Brian` with permissions `all`
7. User `Brian` moves folder `Alice-folder/folder2` to `Carol-folder/folder2`
8. User `Carol` gets the information of the last share as:

```
 curl -u carol:123456 -X GET "http://172.17.0.1/oc/ocs/v1.php/apps/files_sharing/api/v1/shares" -H "OCS_APIREQUEST: true" | xmllint --format -
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  1120  100  1120    0     0   8421      0 --:--:-- --:--:-- --:--:--  8421
<?xml version="1.0"?>
<ocs>
  <meta>
    <status>ok</status>
    <statuscode>100</statuscode>
    <message/>
    <totalitems/>
    <itemsperpage/>
  </meta>
  <data>
    <element>
      <id>22</id>
      <share_type>0</share_type>
      <uid_owner>carol</uid_owner>
      <displayname_owner>carol</displayname_owner>
      <permissions>31</permissions>
      <stime>1591334770</stime>
      <parent/>
      <expiration/>
      <token/>
      <uid_file_owner>carol</uid_file_owner>
      <displayname_file_owner>carol</displayname_file_owner>
      <additional_info_owner/>
      <additional_info_file_owner/>
      <path>/Carol-folder</path>
      <item_type>folder</item_type>
      <mimetype>httpd/unix-directory</mimetype>
      <storage_id>home::carol</storage_id>
      <storage>34</storage>
      <item_source>2147483990</item_source>
      <file_source>2147483990</file_source>
      <file_parent>2147483977</file_parent>
      <file_target>/Carol-folder</file_target>
      <share_with>brian</share_with>
      <share_with_displayname>brian</share_with_displayname>
      <share_with_additional_info/>
      <mail_send>0</mail_send>
      <attributes/>
    </element>
  </data>
</ocs>
```
9. The share id is 22. So when we get the information about the share with `id = 22` as:

```
curl -u carol:123456 -X GET "http://172.17.0.1/oc/ocs/v1.php/apps/files_sharing/api/v1/shares/22" -H "OCS_APIREQUEST: true" | xmllint --format -
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  1120  100  1120    0     0   8615      0 --:--:-- --:--:-- --:--:--  8615
<?xml version="1.0"?>
<ocs>
  <meta>
    <status>ok</status>
    <statuscode>100</statuscode>
    <message/>
    <totalitems/>
    <itemsperpage/>
  </meta>
  <data>
    <element>
      <id>22</id>
      <share_type>0</share_type>
      <uid_owner>carol</uid_owner>
      <displayname_owner>carol</displayname_owner>
      <permissions>31</permissions>
      <stime>1591334770</stime>
      <parent/>
      <expiration/>
      <token/>
      <uid_file_owner>carol</uid_file_owner>
      <displayname_file_owner>carol</displayname_file_owner>
      <additional_info_owner/>
      <additional_info_file_owner/>
      <path>/Carol-folder</path>
      <item_type>folder</item_type>
      <mimetype>httpd/unix-directory</mimetype>
      <storage_id>home::carol</storage_id>
      <storage>34</storage>
      <item_source>2147483990</item_source>
      <file_source>2147483990</file_source>
      <file_parent>2147483977</file_parent>
      <file_target>/Carol-folder</file_target>
      <share_with>brian</share_with>
      <share_with_displayname>brian</share_with_displayname>
      <share_with_additional_info/>
      <mail_send>0</mail_send>
      <attributes/>
    </element>
  </data>
</ocs>
```

## In OCIS:

1. Create users `Alice`, `Brian` and `Carol`
2. User `Alice` creates folder `Alice-folder`
3. User `Alice` creates folder `Alice-folder/folder2`
4. User `Carol` creates folder `Carol-folder`
5. User `Alice` shares folder `Alice-folder` with user `Brian` with permissions `all`
6. User `Carol` shares folder `Carol-folder` with user `Brian` with permissions `all`
7. User `Brian` moves folder `Alice-folder/folder2` to `Carol-folder/folder2`
8. User `Carol` gets the information of the last share as:

```
curl -u carol:123456 -X GET "http://localhost:9140/ocs/v1.php/apps/files_sharing/api/v1/shares" -H "OCS_APIREQUEST: true" | xmllint --format -
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  1178  100  1178    0     0  14365      0 --:--:-- --:--:-- --:--:-- 14365
<?xml version="1.0" encoding="UTF-8"?>
<ocs>
  <meta>
    <status>ok</status>
    <statuscode>100</statuscode>
    <message>OK</message>
  </meta>
  <data>
    <element>
      <id>9867b95b-3f37-4f4a-b1d1-1eafab13bf9a</id>
      <share_type>0</share_type>
      <uid_owner>carol</uid_owner>
      <displayname_owner>User Carol</displayname_owner>
      <permissions>31</permissions>
      <stime>1591342058</stime>
      <parent/>
      <expiration/>
      <token/>
      <uid_file_owner>carol</uid_file_owner>
      <displayname_file_owner>User Carol</displayname_file_owner>
      <additional_info_owner/>
      <additional_info_file_owner/>
      <state>0</state>
      <path>/Carol-folder</path>
      <item_type>folder</item_type>
      <mimetype>httpd/unix-directory</mimetype>
      <storage_id>1284d238-aa92-42ce-bdc4-0b0000009162</storage_id>
      <storage>0</storage>
      <item_source>cca7d265-aff9-4c25-8c9b-b03d42b3e50c</item_source>
      <file_source>cca7d265-aff9-4c25-8c9b-b03d42b3e50c</file_source>
      <file_parent/>
      <file_target>/Carol-folder</file_target>
      <share_with>brian</share_with>
      <share_with_displayname>User Brian</share_with_displayname>
      <share_with_additional_info/>
      <mail_send/>
    </element>
  </data>
</ocs>
```
9. The share id is : 9867b95b-3f37-4f4a-b1d1-1eafab13bf9a. So when we get the information about the share with `id = 9867b95b-3f37-4f4a-b1d1-1eafab13bf9a` as:

```
curl -u carol:123456 -X GET "http://localhost:9140/ocs/v1.php/apps/files_sharing/api/v1/shares/9867" -H "OCS_APIREQUEST: true" | xmllint --format -
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   166  100   166    0     0  11857      0 --:--:-- --:--:-- --:--:-- 11857
<?xml version="1.0" encoding="UTF-8"?>
<ocs>
  <meta>
    <status>error</status>
    <statuscode>400</statuscode>
    <message>Only GET, POST and PUT are allowed</message>
  </meta>
</ocs>
```

```
 curl -u carol:123456 -X GET "http://localhost:9140/ocs/v1.php/apps/files_sharing/api/v1/shares/9867b95b-3f37-4f4a-b1d1-1eafab13bf9a" -H "OCS_APIREQUEST: true" | xmllint --format -
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   166  100   166    0     0  11857      0 --:--:-- --:--:-- --:--:-- 11857
<?xml version="1.0" encoding="UTF-8"?>
<ocs>
  <meta>
    <status>error</status>
    <statuscode>400</statuscode>
    <message>Only GET, POST and PUT are allowed</message>
  </meta>
</ocs>
```

### PR DESCRIPTION
**Pull request by individual-it**
## Description
this PR is on top of #37656, so only review the last commit here

https://github.com/owncloud/ocis-reva/issues/249 is fixed, so find all scenarios tagged with that issue and find out if they should work and enable it or retag with the correct issue

## Related Issue
- Fixes https://github.com/owncloud/ocis-reva/issues/249

## Motivation and Context
clean up

## How Has This Been Tested?
https://github.com/owncloud/ocis-reva/pull/365

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

**Comment by PVince81** [comment]
last commit looks good

so no tests could be enabled I guess?

**Comment by individual-it** [comment]
> last commit looks good
> 
> so no tests could be enabled I guess?

sadly not, too many other issue, specially with sharing

**Review by PVince81 — APPROVED** [review]

File: tests/acceptance/features/apiShareManagementBasic/deleteShare.feature [hunk 1]
```diff
@@ -36,21 +36,35 @@ Feature: sharing
...
+      | 1               | 100             |
+      | 2               | 200             |
+
   Scenario: orphaned shares
     Given using OCS API version "1"
     And these users have been created with default attributes and without skeleton files:
       | username |
       | Alice    |
       | Brian    |
-    And a new browser session for "Alice" has been started
```
Comment (individual-it) [inline tests/acceptance/features/apiShareManagementBasic/deleteShare.feature:45]: that session is not needed and without the test work in OCIS

**Comment by individual-it** [comment]
some tests fail in reva, need investigating

**Comment by PVince81** [comment]
@individual-it ref ?

I've merged https://github.com/owncloud/ocis-reva/pull/362 now as it was green

**Comment by individual-it** [comment]
I've missed tagging/skipping one test in delete shares https://github.com/owncloud/core/pull/37672/files#diff-35aaca9e75175697f0f65df02f6bd140R181 now it should work
retrying tests in ocis-reva https://github.com/owncloud/ocis-reva/pull/365

**Pull request merged**